src/builder.rs

```diff
@@ -302,6 +302,20 @@ impl Builder {
         Ok(())
     }
 
+    /// The net value of the bundle. The value of all spends, minus the value
+    /// of all outputs.
+    pub fn value_balance(&self) -> i64 {
```

Comment:
What is the use case for this method (calculating the value balance from the in-progress builder state)? It would be useful to describe intended use cases in the documentation.